cmd/template/nodepool/runner.go

```diff
@@ -59,6 +59,7 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 			OnDemandBaseCapacity:                r.flag.OnDemandBaseCapacity,
 			OnDemandPercentageAboveBaseCapacity: r.flag.OnDemandPercentageAboveBaseCapacity,
 			Owner:                               r.flag.Owner,
+			ReleaseVersion:                      r.flag.Release,
```

Comment:
We have to add a release flag to the nodepool command in order to recognize when we want a CAPA machinepool.